packages/path_provider/path_provider/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## NEXT
 
+* Updates example app Android compileSdkVersion to 35.
```

Comment:
This changelog should also mention moving the example to include flutter as a declarative dependency.

Reply:
Why are we updating the CHANGELOG at all for this PR? Example-build-only changes are exempt; we generally only mention them if they are about something that is directly relevant to clients of the package (e.g., if a new version of Android requires a new permission to use the code in the package, we might mention adding that permission to the example).

Reply:
Because we are changing the example app which is users facing and should be updated.

Reply:
It's user-facing, but (other than the one file from the example directory shown on pub.dev) it's not directly pub-package-client-facing; if someone `flutter pub upgrade`s and gets a new version of `path_provider`, nothing in this PR is in the code they will import, and that's generally what we consider as the CHANGELOG audience.

We expect people running the example to check out the repo (in theory they could build and run it directly from their pub cache, but they really shouldn't), and the git history is the record of changes for someone interacting with the repo itself.

The things in the CHANGELOG change here are part of what our written policy explicitly exempts, and we also have not historically put this kind of thing in the CHANGELOG, so it seems odd to include it this time.

Reply:
Ok, I misunderstood the policy. @jesswrd sorry for leading you astray but it will make your follow up prs much easier.